request/graphql/schema/types/scalars.go

```diff
@@ -63,3 +64,211 @@ var BlobScalarType = graphql.NewScalar(graphql.ScalarConfig{
 		}
 	},
 })
+
+// BigIntPattern is a regex for validating big int strings
+var BigIntPattern = regexp.MustCompile("^[0-9]+$")
+
+var BigIntScalarType = graphql.NewScalar(graphql.ScalarConfig{
+	Name:        "BigInt",
+	Description: "The `BigInt` scalar type represents an arbitrary precision integer.",
```

Comment:
todo: It is unclear as to what you mean by `arbitrary precision` here, from scanning the code/tests it looks like the user has no way of setting the precision, and that internally everything is just int64?  Please clarify this, same goes for big float.

Please also (PR comment is fine) explain why you chose the name `BigFoo` instead of Int64 or other alternatives.

Reply:
The internal storage for BigInt is a string matching the pattern `^[0-9]+$`. If the number is larger than an `int64` or `uint64` it can be set using a string of the same format.

Reply:
:sweat_smile: Please document that in the description.

That also sounds horribly inefficient.  Why are you doing it like that?

Reply:
Discussed in standup, I'll write my understanding here in case I am unavailable later:

todo: store both types as byte arrays, not strings.

---------------------------

Not discussed in standup, but tidied up a bit from my prior ramblings/thoughts:

todo:  Please don't use the phrase `arbitrary precision` for int/decimal, it only makes sense for float.

suggestion: BigDecimal should also be added (another PR most likely).

suggestion: Break up this PR into one for Int and one for float, but as it is already together I won't push back at all if you don't want to.